### PR TITLE
Cgi/blender 2 80

### PR DESCRIPTION
**Pull request by magdasta**
As for now, Eevee is not going to be supported.

### CODE REVIEW

File: tests/golem/verifier/test_blenderverifier.py [hunk 1]
```diff
@@ -206,7 +206,8 @@ def _create_subtask_info(
             ctd=ComputeTaskDef(
                 deadline=time.time() + 3600,
                 docker_images=[
-                    DockerImage('golemfactory/blender', tag='1.9').to_dict()
+                    DockerImage('golemfactory/blender', tag='blender_2.80')
```
Comment (shadeofblue) [inline tests/golem/verifier/test_blenderverifier.py]: this needs to be updated with a production tag before we merge

Comment (magdasta) [reply tests/golem/verifier/test_blenderverifier.py]: Waiting for docker images updates from @nieznanysprawiciel and @zakaprov

**Comment by shadeofblue** [comment]
@magdasta apart from the docker image, I believe the integration test scenes should be updated (I can do this if needed) and please change the base to `b0.22`

File: apps/images.ini [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-golemfactory/blender blender/resources/images/blender.Dockerfile 1.12.1 blender/resources/images/
-golemfactory/blender_verifier blender/resources/images/blender_verifier.Dockerfile 1.8.2 blender/resources/images/
-golemfactory/blender_nvgpu blender/resources/images/blender_nvgpu.Dockerfile 1.6.1 . apps.core.nvgpu.is_supported
+golemfactory/blender blender/resources/images/blender.Dockerfile 1.13 blender/resources/images/
+golemfactory/blender_verifier blender/resources/images/blender_verifier.Dockerfile 1.9 blender/resources/images/
+golemfactory/blender_nvgpu blender/resources/images/blender_nvgpu.Dockerfile 1.7 . apps.core.nvgpu.is_supported
 golemfactory/dummy dummy/resources/images/Dockerfile 1.4.1 dummy/resources/images
 golemfactory/wasm wasm/resources/images/Dockerfile 0.5.4 wasm/resources/images
-golemfactory/glambda glambda/resources/images/Dockerfile 1.7.1 .
+golemfactory/glambda glambda/resources/images/Dockerfile 1.7.1 .
```
Comment (shadeofblue) [inline apps/images.ini]: @magdasta newline missing?

**Review by shadeofblue — APPROVED** [review]
I guess it's okay, though, I haven't tried looking into the changes to the docker images themselves

**Pull request merged**